storm-core/src/jvm/org/apache/storm/metric/internal/RateTracker.java

```diff
@@ -100,12 +99,6 @@ synchronized double reportRate(long currentTime) {
         return events * 1000.0 / duration;
     }
 
-    public void close() {
-        if (_task != null) {
-            _task.cancel();
```

Comment:
It should have been called when the disruptor queue shuts down, but it wasn't.  Another bug.

https://github.com/abhishekagarwal87/storm/blob/STORM-1693/storm-core/src/jvm/org/apache/storm/utils/DisruptorQueue.java#L392-L400